linera-service/src/server.rs

```diff
@@ -338,6 +338,10 @@ enum ServerCommand {
         #[structopt(long, default_value = "1000")]
         cache_size: usize,
 
+        /// Create a table if one is missing
+        #[structopt(long = "create a database if missing", default_value = "true")]
+        create_if_missing: String,
```

Comment:
Should probably be `bool`.

Reply:
Same problem. I could not find a way to have boolean arguments that are true by default.

Reply:
That's because it's generally not a great idea to do this.

Reply:
The problem with "skip table creation" is that it implies that a table would be created if set to false.
So, I would propose instead `stop if missing table".

Reply:
Not a problem:
`--skip-table-creation-when-missing`
`--prevent-automatic-table-creation`